core/src/main/groovy/noe/rhel/server/workspace/WorkspaceHttpdTomcatsBaseOS.groovy

```diff
@@ -71,10 +101,12 @@ class WorkspaceHttpdTomcatsBaseOS extends WorkspaceMultipleTomcats{
       serverController.getServerById(tomcatId).setHost(originalTomcatHosts.get(tomcatId))
     }
 
-    modJkModules.each() {File module ->
+    moduleFiles.each() { File module ->
       JBFile.delete(module)
     }
 
+    fileStateVault.popAll()
```

Comment:
ok, you use binaries for vault, how did you verified that it is ok, wrt our last discussion?

Reply:
Not binaries, but config files. In this case `mod_cluster.conf`

Reply:
ok, my bad, in that case it is fine.